tests/callbacks/test_finetuning_callback.py

```diff
@@ -357,24 +358,30 @@ def test_callbacks_restore(tmpdir):
     assert len(callback._internal_optimizer_metadata[0]) == 2
 
     # original parameters
-    assert callback._internal_optimizer_metadata[0][0] == {
+    expected_ = {
         "lr": 0.1,
         "momentum": 0,
         "dampening": 0,
         "weight_decay": 0,
         "nesterov": False,
         "params": ["layer.3.weight", "layer.3.bias"],
     }
+    if _TORCH_GREATER_EQUAL_1_11:
+        expected_.update({"maximize": False})
```

Comment:
```suggestion
        expected_["maximize"] = False
```

Reply:
any reason?

Reply:
Your code creates a new dictionary object unnecessarily when we just need to add a key-value

Reply:
are you sure about it? as the update does not return any new object...

Reply:
what you are passing to `update` is a newly created dictionary object

Reply:
ok, now I understand what you mean...